src/cockpit/389-console/src/lib/ldap_editor/lib/utils.jsx

```diff
@@ -998,7 +998,7 @@ export function showCertificate (certificate, showCertCallback) {
         'echo ' + certificate +
     ' | base64 --decode | openssl x509 -inform DER -noout -subject -issuer -dates -serial ;' +
     // ' echo HOST_TIME_GMT=`TZ=GMT date "+%Y-%m-%d %H:%M:%S %Z"` ' // Issue with Firefox and Safari.
-    ' echo HOST_TIME_GMT=`TZ=GMT date`'
+    ' LANG=C; echo HOST_TIME_GMT=`TZ=GMT date`'
```

Comment:
```suggestion
    ' echo HOST_TIME_GMT=`LANG=C TZ=GMT date`'
```
I think this would be more clear since we already pass a variable for `date` to use GMT timezone.

Reply:
Generally speaking, it is better to avoid `VAR=value; cmd` pattern because VAR may not be exported (and therefore may not be set in cmd environment). `export VAR=value; cmd` or `VAR=value cmd` should be used instead.
It is not really an issue in this specific case because if LANG is not set, C is still be used as a default value, but  @vashirov proposal avoid the reader to wonder  whether the code is correct or not! :wink: